### DISCUSSION TITLE
web support?

### DISCUSSION THREAD

**Discussion by nuno** [roninoss/nativewindui#5] [Ideas]
any plans for web support? or is out of the scope of this project? Thanks.

**Comment by ammarfaris** [comment]
I am having the same question for web-support... because i am guessing the end goal should be cross-platform (web and mobile) right? taking the spirit of react-native-reusables and nativewind ... @danstepanov @mrzachnugent

**Comment by mrzachnugent** [comment]
We will have public facing roadmap very soon! (But yes, we have plans for web support 👨‍🍳)

**Reply by ammarfaris** [reply]
Is there a rough date for web support?

**Reply by ammarfaris** [reply]
@mrzachnugent

**Reply by danstepanov** [reply]
There is no rough date
